views/package_format_sdl.dt

```diff
@@ -657,6 +657,9 @@ block body
 		tr
 			td unittest-cov
 			td <code>"unittests" "coverage" "debugMode" "debugInfo"</code>
+		tr
+			td syntax
+			td <code>["syntaxOnly"]</code>
```

Comment:
In SDL brackets aren't required, it should be just `td <code>"syntaxOnly"</code>` instead

Reply:
Thanks, fixed